### PR TITLE
Custom User Model

### PR DESCRIPTION
**Pull request by rominic29**

### CODE REVIEW

File: documentsManager/admin.py [hunk 1]
```diff
@@ -1,7 +1,30 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin as DjangoUserAdmin
+from django.utils.translation import ugettext_lazy as _
 
-from .models import FileDoc, FileType, Event
+from .models import FileDoc, FileType, Event, User
```
Comment (leonardxfce) [inline documentsManager/admin.py]:
```suggestion
from .models import FileDoc, FileType, Event, User
``` 
Pls indent when the imports is too long

File: documentsManager/models.py [hunk 2]
```diff
@@ -4,6 +4,50 @@
...
+
+class UserManager(BaseUserManager):
+
+    use_in_migrations = True
+
+    def _create_user(self, email, password, **extra_fields):
+        if not email:
+            raise ValueError('The given email must be set')
+        email = self.normalize_email(email)
+        # import ipdb; ipdb.set_trace()
```
Comment (leonardxfce) [inline documentsManager/models.py]: Don't push commented code

File: documentsManager/models.py [hunk 3]
```diff
@@ -4,6 +4,50 @@
...
+    def create_user(self, email, password=None, **extra_fields):
+        extra_fields.setdefault('is_staff', False)
+        extra_fields.setdefault('is_superuser', False)
+        return self._create_user(email, password, **extra_fields)
+
+    def create_superuser(self, email, password, **extra_fields):
+        extra_fields.setdefault('is_staff', True)
+        extra_fields.setdefault('is_superuser', True)
+
+        if extra_fields.get('is_staff') is not True:
```
Comment (leonardxfce) [inline documentsManager/models.py]:
```suggestion
        if extra_fields.get('is_staff') is not True:
```
is not True ... is not equals to say is False ??

File: documentsManager/models.py [hunk 4]
```diff
@@ -4,6 +4,50 @@
...
+            raise ValueError('Superuser must have is_staff=True.')
+        if extra_fields.get('is_superuser') is not True:
```
Comment (leonardxfce) [inline documentsManager/models.py]:
```suggestion
        if extra_fields.get('is_superuser') is not True:
```
Same here.

File: documentsManager/templates/home.html [hunk 5]
```diff
@@ -197,6 +190,7 @@ <h1 class="eds-empty-state__title eds-text--center eds-text-hs eds-text-color--g
 	        </div>
 	    </div>
 	</div>
+	<input type="hidden" id="eb_user" value="{{is_eb_user}}">
```
Comment (leonardxfce) [inline documentsManager/templates/home.html]:
```suggestion
	<input type="hidden" id="eb_user" value="{{is_eb_user}}">
```
Pls add some spaces before and after the variable {{ is_eb_user }}

File: documentsManager/admin.py (hunk 1 above)
Comment (rominic29) [reply documentsManager/admin.py]: Ok.

File: documentsManager/models.py (hunk 2 above)
Comment (rominic29) [reply documentsManager/models.py]: Removed.

File: documentsManager/templates/home.html (hunk 5 above)
Comment (rominic29) [reply documentsManager/templates/home.html]: Ok.

File: documentsManager/models.py (hunk 3 above)
Comment (rominic29) [reply documentsManager/models.py]: In this case, I need to specifically corroborate that it is not True. In that field you can enter something different to False or True.

File: documentsManager/models.py (hunk 4 above)
Comment (rominic29) [reply documentsManager/models.py]: In this case, I need to specifically corroborate that it is not True. In that field you can enter something different to False or True.

**Pull request merged**